Planet/CountryPickerViewController.swift

```diff
@@ -20,7 +20,13 @@ public class CountryPickerViewController: UITableViewController {
     public var showsCancelButton = true
     
     fileprivate var countryDataSource = CountryDataSource()
-    
```

Comment:
This should then also be appropriately named `countryCodes` like the parameter to `CountryDataSource` and it should probably also have a default of `Locale.isoRegionCodes`, that is all the country codes.